service/src/main/java/org/cbioportal/genome_nexus/service/annotation/VariantAnnotationInputFormat.java

```diff
@@ -0,0 +1,8 @@
+
+package org.cbioportal.genome_nexus.service.annotation;
+
+public enum VariantAnnotationInputFormat {
```

Comment:
We may not even need this `enum`, if we implement a different service for each different data format, and decide which service to call at the controller level.